thrift/contentatom.thrift

```diff
@@ -0,0 +1,46 @@
+include "atoms/quiz.thrift"
+include "shared.thrift"
+
+typedef string ContentAtomID
+
+enum AtomType { QUIZ }
+
+struct Atom {
+  1: required ContentAtomID id
+  2: required AtomType atomType
+  3: required list<string> labels // required, but may be empty
+  4: required string defaultBody
```

Comment:
I would rather call it `defaultHtml` than `defaultBody` but this is just a preference.